bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LedgerDirsManager.java

```diff
@@ -176,19 +176,10 @@ public boolean hasWritableLedgerDirs() {
             return writableLedgerDirectories;
         }
 
-        // If Force GC is not allowed under no space
-        if (!forceGCAllowWhenNoSpace) {
```

Comment:
any reason for why this check is removed?

Reply:
1) `forceGCAllowWhenNoSpace` is controlling the gc behavior, it controls whether gc/compaction
2) this setting is replaced by `minUsableSizeForEntryLogCreation`. because not only gc/compaction will create the entry log files on readonly, during journal replays it would still need create entry log files. This allows we disableForceGC when no space but we can still allow bookie start up in readonly mode to replay journals if there is enough space.